packages/reverse-proxy-service/src/couchdb/resolver.ts

```diff
@@ -1,7 +1,7 @@
 import { createHmac } from "crypto";
 import { Request, Response, NextFunction } from "express";
 import { createProxyMiddleware } from "http-proxy-middleware";
-import { COUCHDB_HOST, COUCHDB_SECRET } from "../config/env";
+import { COUCHDB_HOST, COUCHDB_SECRET } from "../setup/env";
```

Comment:
Single quotes should be used

Reply:
Fixed. Added a `.prettierrc` to overwrite the defaults.